Apps/Sandcastle/gallery/EPSG2039.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no">
+    <meta name="description" content="Demonstration of Proj4JS projections.">
+    <meta name="cesium-sandcastle-labels" content="Showcases">
+    <title>Cesium Demo</title>
+    <script type="text/javascript" src="../Sandcastle-header.js"></script>
+    <script type="text/javascript" src="../../../ThirdParty/requirejs-2.1.20/require.js"></script>
+    <script type="text/javascript">
+        if(typeof require === "function") {
+            require.config({
+                baseUrl : '../../../Source',
+                waitSeconds : 120
+            });
+        }
+    </script>
+</head>
+<body class="sandcastle-loading" data-sandcastle-bucket="bucket-requirejs.html">
+<style>
+    @import url(../templates/bucket.css);
+</style>
+<div id="cesiumContainer" class="fullSize"></div>
+<div id="loadingOverlay"><h1>Loading...</h1></div>
+<div id="toolbar"></div>
+<script id="cesium_sandcastle_script">
+function startup(Cesium) {
+    'use strict';
+//Sandcastle_Begin
+var viewer = new Cesium.Viewer('cesiumContainer', {
+    sceneMode : Cesium.SceneMode.COLUMBUS_VIEW,
+    mapProjection : new Cesium.Proj4Projection('+proj=tmerc +lat_0=31.73439361111111 +lon_0=35.20451694444445 +k=1.0000067 +x_0=219529.584 +y_0=626907.39 +ellps=GRS80 +towgs84=-48,55,52,0,0,0,0 +units=m +no_defs '),
+    terrainProvider: Cesium.createWorldTerrain({
+        requestVertexNormals : true,
+        requestWaterMask : true
+    })
+});
+
+var epsg2039Bounds = Cesium.Rectangle.fromDegrees(34.2200, 29.4900, 35.6800, 33.2700);
+
+viewer.scene.globe.tileLimitRectangle = epsg2039Bounds;
```

Comment:
What is the overall plan?  Is this something that will only apply to the globe?  Or do we have to add a property like this to anything that can render?  Or just a few key primitives?

Also I don't know about the name - it seems to leak an implementation detail - think of it from the end users' perspective - what would they think this should be called?

Reply:
Right now this only applies to the globe and only in 2D and in CV, but getting it to work in 3D won't be hard if we decide it's useful there despite the implicit lack of a map projection.

Having this work for primitives could be tricky, and I thought it might not be worthwhile in the general case. I imagine that anyone working in a super-local projection won't be placing stuff very far outside the bounds anyway. One exception I can think of is an app showing global-scale paths passing through a local area, with the ability to show the local area in 2D but the full paths in 3D. You could build that without rectangle limiter support on Primitives, though, it'd just take some extra user code.

For the name, maybe `geographicLimitRectangle`?